src/main/java/org/influxdb/impl/InfluxDBService.java

```diff
@@ -37,35 +36,36 @@
    * @param consistency optional The write consistency level required for the write to succeed.
    *                             Can be one of one, any, all, quorum. Defaults to all.
    */
-  @POST("/write")
+  @POST("write")
   public Call<ResponseBody> writePoints(@Query(U) String username,
-      @Query(P) String password, @Query(DB) String database,
-      @Query(RP) String retentionPolicy, @Query(PRECISION) String precision,
-      @Query(CONSISTENCY) String consistency, @Body RequestBody batchPoints);
+                                        @Query(P) String password, @Query(DB) String database,
```

Comment:
same here, and all other parts down here.

Reply:
Noticed, that codestyle check stops building the project when there are lines longer tan 120 characters